soh/soh/Enhancements/crowd-control/CrowdControlTypes.h

```diff
@@ -0,0 +1,302 @@
+typedef enum {
+
+    // Spawn Enemies and Objects
+    effectSpawnCuccoStorm,
+    effectSpawnLitBomb,
+    effectSpawnExplosion,
```

Comment:
Small nit pick, but the casing of these enums differs from how we've done all our other enums. It is self contained to just the crowd control files though, so I'm not super picky if you feel you don't want to change the casing here.

Reply:
Oh, you mean the 'e' being lowercase?

Reply:
The change you made is better (`PascalCase` now vs `camelCase`), I was more so referring to our other enums being `ALL_CAPS`.

Reply:
I don't know how I completely blanked on that, lol. That uhh, would be a decent bit of work to fix. I guess it depends on how much we care about having them all caps?

Reply:
This was discussed on discord, but just to have it documented here too: We might move towards the same formatting as Google and Apple are using. That means defines and macros will be ALL_CAPS, enums will be kPascalCase.

We'll see in the future if we want to spend the time to convert existing enums as well.